app/src/pages/patientSearchPage/PatientSearchPage.tsx

```diff
@@ -115,6 +115,7 @@ function PatientSearchPage() {
                     <Input
                         id="nhs-number-input"
                         data-testid="nhs-number-input"
+                        className="nhsuk-input--width-10"
```

Comment:
Would it be worth having something like `maxlength="12"` limiting it to 12 chars